lightning/Cargo.toml

```diff
@@ -21,11 +21,11 @@ max_level_debug = []
 
 [dependencies]
 bitcoin = "0.23"
+rand = "0.4"
```

Comment:
#641 just removed the rand dev-dependency, but we can't rely on it in regular usage anyway - it certainly doesn't support all platforms that may want to run RL (incl, eg, hardware wallets).

Reply:
So what's our approach to randomness then? In this context it's desirable for routing privacy.